mettle/worker/worker.go

```diff
@@ -915,6 +924,27 @@ func (w *worker) update(stage pb.ExecutionStage_Value, response *pb.ExecuteRespo
 	return common.PublishWithOrderingKey(ctx, w.responses, body, w.actionDigest.Hash, w.name)
 }
 
+// periodicallyPushMetrics will push this worker's metrics to the gateway every 5 minutes.
+func (w *worker) periodicallyPushMetrics() {
+	if w.promGatewayURL != "" {
+		// Set up the metrics
+		var pushers []*push.Pusher
```

Comment:
Can you not have just one Pusher? From the docs it looks like you should be able to do `.Collector(metric1).Collector(metric2)` to jam them all into the same one.

Reply:
Yep I think you're right, for some reason I thought they all needed different job names. Updated now to use one pusher, thanks again :)